### DISCUSSION TITLE
Should sibling's parent my implied parent?

### DISCUSSION THREAD

**Discussion by ooker777** [SkepticMystic/breadcrumbs#323] [Ideas]
![image](https://user-images.githubusercontent.com/6695517/152635364-de8407d1-0c10-4dd3-9197-d6d83baabf4b.png)

**Comment by SkepticMystic** [comment]
Hahah I literally just added this feature 😋 
https://github.com/SkepticMystic/breadcrumbs/issues/301

**Reply by ooker777** [reply]
hahaha cool

**Reply by HananoshikaYomaru** [reply]
It is an old problem. Glad that it is fixed.

**Comment by michaelpporter** [comment]
Yes — in v4 this is supported as an opt-in **transitive rule**.

Add this under **Settings → Breadcrumbs → Implied relations** (the bulk-add box takes `[field-one, field-two] -> close-field` syntax):

```
[same, up] -> up
```

It reads as "if X is `same` as Y, and Y is `up` to Z, then X is implied `up` to Z" — i.e. a sibling inherits its sibling's parent. The implied edge then shows in all views (Matrix, Tree, trail).

It's not enabled by default (v4 ships only single-hop close pairs like `up`↔`down`, `same`↔`same`), so you add exactly the chains you want. Same answer as #247. Closing as resolved.
